app/controllers/application_controller.rb

```diff
@@ -3,7 +3,22 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
-  def  aaa
-      1 + 1
+  def  aaa (a, b   , c,d,e,f    ,    g)
+        1 + 1
+        puts "asdfas dfas dfas dfasd fasd fasdf asd fasd fasd fasd asd fasdf asdfas dfas dfas dfasd fasd fasd fasd f"
+      { one: 1, two: 2 }
+  case
+    when 1 == 2
+      puts 'Not again!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.